src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final Random RANDOM = new Random();
+    static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
+    static final Ball BALL = new Ball();
+    static final int BOUND = 100;
```

Comment:
don't forget about access modifiers
random and color supplier shouldn't be static
ball shouldn't be a field, it's redundant. It's ok to create a new ball instance each time you want to get a random ball